File: removebg.cs
using System;
using System.Drawing;
using System.Drawing.Imaging;

class Program {
    static void Main() {
        try {
            string inPath = @"C:\Users\einst\.gemini\antigravity\brain\df108df2-b693-44c3-8fb9-534af7f90448\media__1773097916120.png";
            string outPath1 = @"c:\SOMAR\AG\leadbroker-ai\public\logo-domvia.png";
            string outPath2 = @"c:\SOMAR\AG\leadbroker-ai\public\icon.png";
            
            using (Bitmap img = new Bitmap(inPath)) {
                int width = img.Width;
                int height = img.Height;

                for (int x = 0; x < width; x++) {
                    for (int y = 0; y < height; y++) {
                        Color c = img.GetPixel(x, y);
                        if (c.R > 240 && c.G > 240 && c.B > 240) {
                            img.SetPixel(x, y, Color.Transparent);
                        }
                    }
                }
                img.Save(outPath1, ImageFormat.Png);
                img.Save(outPath2, ImageFormat.Png);
                Console.WriteLine("Background removed successfully");
            }
        } catch (Exception ex) {
            Console.WriteLine("Error: " + ex.Message);
        }
    }
}

File: remove-bg.js
const Jimp = require("jimp");

async function removeWhite() {
    try {
        const image = await Jimp.read("C:\\Users\\einst\\.gemini\\antigravity\\brain\\df108df2-b693-44c3-8fb9-534af7f90448\\domvia_logo_d_spark_1773097374842.png");

        // Scan the image pixels
        image.scan(0, 0, image.bitmap.width, image.bitmap.height, function (x, y, idx) {
            const r = this.bitmap.data[idx + 0];
            const g = this.bitmap.data[idx + 1];
            const b = this.bitmap.data[idx + 2];

            // Se for branco ou cinza bem claro
            if (r > 245 && g > 245 && b > 245) {
                this.bitmap.data[idx + 3] = 0; // Transparente
            }
        });

        await image.writeAsync("c:\\SOMAR\\AG\\leadbroker-ai\\public\\logo-domvia.png");
        await image.writeAsync("c:\\SOMAR\\AG\\leadbroker-ai\\public\\icon.png");

        console.log("Transparência aplicada com sucesso!");
    } catch (e) {
        console.error("Erro processando imagem:", e);
    }
}

removeWhite();
